src/fixup.js

```diff
@@ -139,4 +140,45 @@
     }
   }
 
+  /* Deprecation warning */
+  var request = new XMLHttpRequest();
+
+  request.open('GET', '//www.w3.org/TR/tr-outdated-spec');
+  request.onload = function() {
+    if (request.status >= 200 && request.status < 400) {
+      var currentSpec = JSON.parse(request.responseText);
+      if (currentSpec) {
+        document.body.classList.add('outdated-spec');
+        var node = document.createElement("p");
+        node.classList.add("outdated-warning");
+
+        var warning = '<strong>' + currentSpec.header + '</strong><div>' + currentSpec.warning + '<a id="outdated-note" href="' + currentSpec.latestUrl + '"> ' + currentSpec.latestUrl + '</a>.</div><input onclick="collapseWarning(false)" type="button" value="&#9662; collapse">';
```

Comment:
This markup doesn't seem correct to... if you put a `div` in there, it will auto-close the `<p>`? Maybe you want a `span` that is displayed as an block instead?